fs/src/io_backend.rs

```diff
@@ -0,0 +1,109 @@
+use std::io;
+
+#[cfg(target_os = "linux")]
+const SQPOLL_IDLE_WAIT_TIME: u32 = 50;
+
+/// State used by IO utilities for managing shared resources.
```

Comment:
Should probably called IoState or something then? 

Backend make me think that there are multiple alternative backends

Reply:
changed to `IoSetupState`